### ISSUE TITLE
i face too  many issues during installing MSYS2 

### ISSUE THREAD

**Issue by coook237** [msys2/MSYS2-packages#2795]
![Screenshot 2022-01-07 171516](https://user-images.githubusercontent.com/97048850/148632467-54617ea1-83b3-4d46-97e6-4c659654b48c.png)

if anyone can help about it i am really thankfull to u 
i will wait for your response.
thank you

**Comment by 1480c1** [comment]
It seems you are still able to update after pressing Y and enter? Perhaps you just need to update pacman?

**Comment by cbrt64** [comment]
TL;DR Spurious/unnecessary error messages, just ignore. 😃

I've noticed this behavior ever since Arch added parallel downloads to pacman. When pacman fails to download some file from one server, it falls back to another, and the download completes as shown, but pacman prints an error message for the failed connection anyway. (The cause of dropped connections is a different matter, and something I wouldn't know much about.)

In any case, if downloading the db files were to actually fail, they wouldn't get to 100%, and pacman would not ask you to continue as it does. When this happens to me I press `Y`, then onward.

**Comment by stahta01** [comment]
I have a poor internet connection.

`pacman -Sy --disable-download-timeout`

I use the above to sync the information.

Tim S.
